crates/nu-command/Cargo.toml

```diff
@@ -79,7 +79,7 @@ sha2 = "0.10.0"
 # Disable default features b/c the default features build Git (very slow to compile)
 shadow-rs = { version = "0.16.1", default-features = false }
 strip-ansi-escapes = "0.1.1"
-sysinfo = "0.26.2"
+sysinfo = { default-features = false, version = "0.26.2" }
```

Comment:
https://docs.rs/sysinfo/latest/sysinfo/#usage 

Here we might have to look into whether we would regress on the performance of `sys` or `ps` if we disable multithreading.

Reply:
I tested benchmarking nu `ps` command on my windows device and saw no major difference between the multithreading and non multithreading version. I don't know how other OS may be affected. Should we remove this one? to ensure no regression in this area?

Reply:
> Should we remove this one? to ensure no regression in this area?

Yes, please leave multithreading on in `sysinfo`.

Reply:
Not depending on `rayon` here would allow for some earlier compilation of `sysinfo` which depending on the config might allow more parallel compilation.
But out of laziness I only compared `benchmark { ps -l }` on my WSL2-VM as it was quickest to recompile there for build timings but the number of processes on that is limited to maybe not see a drastic differences in scaling.
I think @rgwood you were fixing the recent problems with `ps` so you have the most recent insight into that.

Reply:
I don't currently have any numbers but I think that if multithreading is the default we should probably just leave it on.

At the very least I'd want to see some numbers indicating that disabling multithreading meaningfully improves compile times.

Reply:
I added back this in my latest update.

Reply:
@FilipAndersson245 I think you missed this line

Reply:
Ops, my bad, fixed bdc7c1e768ba9dc325813d7b0019cb114e2a82a9.